conf/config.json

```diff
@@ -17,6 +21,6 @@
         "https://matrix-client.matrix.org": false
     },
     "embedded_pages": {
-        "login_for_welcome": __LOGIN_FOR_WELCOME__
+        "login_for_welcome": "__LOGIN_FOR_WELCOME__"
```

Comment:
Hmmmm, it this really needed ? o_O

It looks like the value is meant to be `true` which I would guess should be used as an actual boolean and not the string `"true"` ?

Reply:
`true` 😅